From 58085dc56502e91361dbf2a731436b609c6e9cc7 Mon Sep 17 00:00:00 2001
From: Jingchang Lu <jingchang.lu@freescale.com>
Date: Mon, 11 Aug 2014 13:56:35 +0800
Subject: [PATCH 007/128] arm: dts: ls1021a: change duart compatible to
 "fsl,16550-FIFO64"

this patch change the duart compatible to 64-byte FIFO mode.

Signed-off-by: Jingchang Lu <jingchang.lu@freescale.com>
Change-Id: I4d306671daed4262a6f354a3507304d82468c41d
Reviewed-on: http://git.am.freescale.net:8181/17835
Tested-by: Review Code-CDREVIEW <CDREVIEW@freescale.com>
Reviewed-by: Huan Wang <alison.wang@freescale.com>
Reviewed-by: Li Xiubo <Li.Xiubo@freescale.com>
Reviewed-by: Zhengxiong Jin <Jason.Jin@freescale.com>
[Xulin: Original patch taken from
QorIQ-SDK-V1.7-SOURCE-20141218-yocto_RDS_20150206.iso]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 arch/arm/boot/dts/ls1021a.dtsi |   16 ++++++++--------
 1 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/arch/arm/boot/dts/ls1021a.dtsi b/arch/arm/boot/dts/ls1021a.dtsi
index 91aa312..e1076fa 100644
--- a/arch/arm/boot/dts/ls1021a.dtsi
+++ b/arch/arm/boot/dts/ls1021a.dtsi
@@ -278,38 +278,38 @@
 		};
 
 		duart0: serial@21c0500 {
-			compatible = "fsl,ns16550", "ns16550a";
+			compatible = "fsl,16550-FIFO64";
 			reg = <0x0 0x21c0500 0x0 0x100>;
 			interrupts = <GIC_SPI 86 IRQ_TYPE_LEVEL_HIGH>;
 			clock-frequency = <0>;
-			fifo-size = <8>;
+			fifo-size = <63>;
 			status = "disabled";
 		};
 
 		duart1: serial@21c0600 {
-			compatible = "fsl,ns16550", "ns16550a";
+			compatible = "fsl,16550-FIFO64";
 			reg = <0x0 0x21c0600 0x0 0x100>;
 			interrupts = <GIC_SPI 86 IRQ_TYPE_LEVEL_HIGH>;
 			clock-frequency = <0>;
-			fifo-size = <8>;
+			fifo-size = <63>;
 			status = "disabled";
 		};
 
 		duart2: serial@21d0500 {
-			compatible = "fsl,ns16550", "ns16550a";
+			compatible = "fsl,16550-FIFO64";
 			reg = <0x0 0x21d0500 0x0 0x100>;
 			interrupts = <GIC_SPI 87 IRQ_TYPE_LEVEL_HIGH>;
 			clock-frequency = <0>;
-			fifo-size = <8>;
+			fifo-size = <63>;
 			status = "disabled";
 		};
 
 		duart3: serial@21d0600 {
-			compatible = "fsl,ns16550", "ns16550a";
+			compatible = "fsl,16550-FIFO64";
 			reg = <0x0 0x21d0600 0x0 0x100>;
 			interrupts = <GIC_SPI 87 IRQ_TYPE_LEVEL_HIGH>;
 			clock-frequency = <0>;
-			fifo-size = <8>;
+			fifo-size = <63>;
 			status = "disabled";
 		};
 
-- 
1.7.5.4

